yuhow/pstake.py

```diff
@@ -1,806 +0,0 @@
-#!/usr/bin/env python
```

Comment:
Yes. I agree with your point of view. There is no any specific reason. In the past, I used ppt to make fancy diagram and converted it into PDF file. Also I used PdfLatex to compile my tex file.
Anyway, I will start to use PSTricks for making diagram. It's now that hard to do that. Once I have PDF file, some software can help me to convert it into EPS format.